### DISCUSSION TITLE
How can I use libvips on QT or Can I complier libvips for QT

### DISCUSSION THREAD

**Discussion by RLYEric** [libvips/libvips#3104] [Help wanted]
Hello,
    How can I use libvips on QT or Can I complier libvips for QT?

**Comment by jcupitt** [comment]
Hi @RLYEric, sure, it's just a library, you can use it like any other.

**Reply by RLYEric** [reply]
Hi @jcupitt you means that I just to use QT to complier these source codes to a library  and I also can direct to use these for import these codes into the QT project.

**Reply by jcupitt** [reply]
The details depend on your platform, your compiler, which version of libvips, and where you got the libvips binary from. But generally it's like any other library.

**Reply by RLYEric** [reply]
I understand, thanks!
